programs/keeper/Keeper.cpp

```diff
@@ -623,6 +624,19 @@ try
     buildLoggers(config(), logger());
     main_config_reloader->start();
 
+    std::optional<CgroupsMemoryUsageObserver> observer;
+    auto cgroups_memory_observer_wait_time = config().getUInt64("keeper_server.cgroups_memory_observer_wait_time", 1);
```

Comment:
1 second seems very aggressive. The default for server is 15 sec.

Reply:
I guess the setting should be documented in the user docs somewhere?

Reply:
yes, docks for keeper is in `clickhouse-docs` repo ... so I will add it after merged